resources/views/feedbackboards/mostvoted.blade.php

```diff
@@ -2,7 +2,7 @@
 
     <div class="card mb-3">
         <div class="card-header bg-dark text-white">
-            Most upvoted idea
+            Most upvoted recent {{ strtolower(str_singular($category->title)) }}
```

Comment:
```suggestion
            Most upvoted recent {{ Str::lower(Str::singular($category->title)) }}
```

I think those methods are the 'old' methods, these are used in the docs of laravel:
https://laravel.com/docs/10.x/strings#method-str-singular

Reply:
Yes, we use a Laravel first party library that adds backwards compatibility for these functions so we don't have to replace all these method calls in the code. We should go over the whole codebase some time to fix this.